edx_django_utils/plugins/docs/decisions/0003-plugin-contexts.rst

```diff
@@ -0,0 +1,88 @@
+Plugin Contexts
+---------------
+
+Status
+======
+Draft
+
+Context
+=======
+edx-platform contains a plugin system (https://github.com/edx/edx-platform/tree/master/openedx/core/djangoapps/plugins) which allows new Django apps to be installed inside the LMS and Studio without requiring the LMS/Studio to know about them. This is what enables us to move to a small and extensible core. While we had the ability to add settings, URLs, and signal handlers in our plugins, there wasn't any way for a plugin to affect the commonly used pages that the core was delivering. Thus a plugin couldn't change any details on the dashboard, courseware, or any other rendered page that the platform delivered.
```

Comment:
Add a note above this that stands out and states that this ADR existed with this functionality in edx-platform, before it was moved to edx-django-utils.  You can state that this is still written with edx-platform language, but is now available for all IDAs (I presume?).